src/main/java/com/groupinfo5ltd/tastyManagement/service/impl/VendeurService.java

```diff
@@ -0,0 +1,69 @@
+package com.groupinfo5ltd.tastyManagement.service.impl;
+
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.groupinfo5ltd.tastyManagement.entity.Vendeur;
+import com.groupinfo5ltd.tastyManagement.repository.IVendeurRepository;
+import com.groupinfo5ltd.tastyManagement.service.IVendeurService;
+
+public class VendeurService implements IVendeurService {
+
+	private static final Logger log = LoggerFactory.getLogger(VendeurService.class);
+	@Autowired
+	IVendeurRepository vendeurRepository;
+
+	@Override
+	public Vendeur ajouterVendeur(Vendeur vendeur) {
+		Vendeur vendeurToCreate = null;
+		try {
+			vendeurToCreate = vendeurRepository.save(vendeur);
+			log.info("CREATE A new Vendeur" + vendeur.toString());
+		} catch (Exception e) {
+			log.error("CREATE A new Vendeur" + vendeur.toString());
```

Comment:
f l message dial had log 7sn ktbou
`log.error("UNABLE to CREATE A new vendeur Caused by: " + e.toString())`

Reply:
rah dert error donc ghatban error :""CREATE A new Vendeur" + vendeur.toString()"

Reply:
ahh, 3ndk s7